src/main/java/lotto/domain/Lottos.java

```diff
@@ -20,12 +51,44 @@ private void generateLottoGroup(int count) {
         }
     }
 
-    public static String findResult(Map<Rank, Integer> countByRank) {
-        Money.findEarning(countByRank);
-        return Money.findEarningRate();
+    public ArrayList<Rank> drawLotto(WinningLotto winningLotto) {
+        ArrayList<Rank> wins = new ArrayList<>();
+        for (Lotto lotto : lottoGroup) {
+            Rank rank = winningLotto.findRank(lotto);
+            wins.add(rank);
+        }
+        return wins;
+    }
+
+    public ArrayList<ArrayList<Integer>> changeFormat() {
+        ArrayList<ArrayList<Integer>> displayFormat = new ArrayList<>();
+        for (Lotto lotto : lottoGroup) {
+            displayFormat.add(lotto.changeToList());
+        }
+        return displayFormat;
+    }
+
+    public void generateManualLottos(String manualCount, String manualNumberInput) {
+        List<Lotto> manualLottos = new ArrayList<>();
+        for (int i = 0; i < Integer.parseInt(manualCount); i++) {
+            manualLottos.add(Lotto.changeStringToLotto(manualNumberInput));
+        }
+        lottoGroup.addAll(manualLottos);
+    }
```

Comment:
이렇게 List<String> 으로 수동 번호들을 받고 그 String들을 Lotto에게 전달해서 Lotto 생성과 Lottos 생성이 동시에 일어났으면 했습니다 :) 

```suggestion
    public void generateManualLottos(List<String> manualNumberInput) {
        this.lottoGroup = new ArrayList<>();
        for (int i = 0; i < Integer.parseInt(manualCount); i++) {
            this.lottoGroup.add(Lotto.changeStringToLotto(manualNumberInput));
        }
    }
```